docs-compose/overwatch_docs/content/DataEngineer/Pipeline_Management.md

```diff
@@ -120,6 +104,47 @@ For the developers -- This output is created as a DataSet via the Case Class
 | Pipeline_SnapTS      | timestamp | Timestamp type of the pipeline snapshot time (server time)                                                                                                                                                                          |
 | Overwatch_RunID      | string    | GUID for the overwatch run                                                                                                                                                                                                          |
 
+## The PipReport View
+A *pipReport* view has been created atop the pipeline_report table to simplify reviewing the historical runs. The
+query most commonly used is below. If you are asked for the pipReport, please provide the output from the following
+
+**SCALA**
+```scala
+table("overwatch_etl.pipReport")
+  .filter('organization_id === "<workspace_id>") // if you want to see the pipReport for only a specific workspace
+  .orderBy('Pipeline_SnapTS.desc)
+```
+**SQL**
+```sql
+select * from overwatch_etl.pipReport
+    where organization_id = '<workspace_id>' -- if you want to see the pipReport for only a specific workspace
+    order by Pipeline_SnapTS desc
+```
+
+The structure of the table is outlined [below](#pipreport-structure).
+
+### PipReport Structure
+[**SAMPLE**](/assets/TableSamples/pipreport.tab)
+
+**KEY** -- organization_id + moduleID + Overwatch_RunID
```

Comment:
Isn't workspace_name also part of the key?

Reply:
no workspace_name is not part of the key